### DISCUSSION TITLE
page.find_tables  how to extract words in table cell

### DISCUSSION THREAD

**Discussion by wangqiangJN** [pymupdf/PyMuPDF#3768] [Looking for help]
**Is your feature request related to a problem? Please describe.**
pymupdf version 1.24.9

1 I want to parse my pdf , my pdf contains tables and other text.
2.when I use page. find_tables ,it can extract text in cell , but I find when cell has multi words ,as
     example cell
 _price:520                         people:bob_
     expect result by words :
     price
     520
     people
     bob
    but results now 
    price:520 people:bob
3. so i want to split table cell content by words and   get bbox, here have any function method and solution?

**Comment by JorjMcKie** [comment]
This feature is already there:
Simply extract the text using each table cell as clip rectangle like this `cell_words = page.get_text("words", clip=cell)`.

A minor issue may arise when the table has very narrow cell borders. Then the table finder might identify cell content that technically is not completely inside a cell. The general text extraction is very strict and will discard everything not completely inside the rectangle.
If this happens, just enlarge the cell a bit in each direction when you use it as a clip: I.e. use `clip=pymupdf.Rect(cell)+(-3,-3,3,3)`.
